scripts/workspace_deploy.py

```diff
@@ -81,13 +81,20 @@
     version = getattr(config, 'conversation_version')
     workspacesUrl += '?version=' + version
 
+    # Make sure intents, entities and dialog_nodes fields are present even when empty to make sure the old items get erased.
+    if 'intents' not in workspace: workspace['intents'] = []
+    if 'entities' not in workspace: workspace['entities'] = []
+    if 'dialog_nodes' not in workspace: workspace['dialog_nodes'] = []
+
```

Comment:
Make sure the whole workspace is replaced by new content. If some of these fields is missing, the corresponding content would stay.

Reply:
The reason for this change were complaints about old content hanging in the workspace. Later, I noticed some renamed intents staying in the generated folder. This indicates the problem might be rather in clearing the generated content. I will look into it.

Reply:
Not for now, can be done in #236